mlflow/pipelines/dag_help_strings.py

```diff
@@ -262,7 +262,7 @@ def weighted_mean_squared_error(
 )
 
 PREDICT_STEP = format_help_string(
-    """The 'predict' step uses the model registered by the 'register' step to score the ingested dataset produced by the 'ingest_scoring' step and writes the resulting dataset to the specified output format and location. Namely, it uses the latest version of the registered model specified by the `model_name` attribute of the pipeline.yaml 'register' step definition. To fix a specific model for use in the 'predict' step, provide its model URI as the 'model_uri' attribute of the pipeline.yaml 'predict' step definition. An example pipeline.yaml 'predict' step definition is shown below.
+    """The 'predict' step uses the model registered by the 'register' step to score the ingested dataset produced by the 'ingest_scoring' step and writes the resulting dataset to the specified output format and location. To get model for scoring, it reads the register step model version artifact. If the register step was cleared, it uses the latest version of the registered model specified by the `model_name` attribute of the pipeline.yaml 'register' step definition. To fix a specific model for use in the 'predict' step, provide its model URI as the 'model_uri' attribute of the pipeline.yaml 'predict' step definition. An example pipeline.yaml 'predict' step definition is shown below.
```

Comment:
One big risk of this change, is that it adds a direct dependency to the training DAG. Given its intended usage, it seems fine to me. But in that case, we may need to add an explicit dep here in the graph, which means updating this file as well: `mlflow/pipelines/resources/pipeline_dag_template.html`

Reply:
No let's not add the execution deps. You can merge the PR as is. We can discuss how these deps fit with each other based on how users actually use the batch inference functionality.